service/descriptors/ModuleDescriptor-template.json

```diff
@@ -422,14 +490,16 @@
       { "name": "DB_QUERYTIMEOUT", "value": "60000" },
       { "name": "DB_CHARSET", "value": "UTF-8" },
       { "name": "DB_MAXPOOLSIZE", "value": "16" },
-      { "name": "EVENT_UPLOADS_PATH", "value": "events", "description": "The ActiveMQ event uploads path." },
-      { "name": "EVENT_QUEUE_NAME", "value": "event.queue", "description": "The ActiveMQ event queue name." },
-      { "name": "OKAPI_CAMUNDA_BASEPATH", "value": "/", "description": "The context path, or base path of Mod-Camunda." },
-      { "name": "OKAPI_CAMUNDA_RESTPATH", "value": "/rest", "description": "The path Mod-Camunda uses for REST requests." },
+      { "name": "KAFKA_HOST", "value": "kafka" },
+      { "name": "KAFKA_PORT", "value": "9092" },
+      { "name": "KAFKA_SECURITY_PROTOCOL", "value": "PLAINTEXT" },
+      { "name": "KAFKA_SSL_KEYSTORE_LOCATION", "value": "" },
+      { "name": "KAFKA_SSL_KEYSTORE_PASSWORD", "value": "" },
+      { "name": "KAFKA_SSL_TRUSTSTORE_LOCATION", "value": "" },
+      { "name": "KAFKA_SSL_TRUSTSTORE_PASSWORD", "value": "" },
```

Comment:
The `description` fields are required.
Please add (non-empty) descriptions.

Reply:
Why not create an issue to make them all consistent?

Reply:
Added descriptions and env section to readme.